documentation/book/assembly-kafka-entity-operator.adoc

```diff
@@ -0,0 +1,36 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-entity-operator-{context}']
+
+= Entity Operator
+
+Entity Operator is responsible for managing different entities in a running Kafka cluster.
+The currently supported entities are:
+
+* Kafka topics
```

Comment:
Let's turn this into a defintion list:

* Kafka topics
* Kafka users
 Kafka topics are managed by the Topic Operator.
Kafka users' credentials and access control rules are managed by the User Operator.

TO:

* Kafka topics:: managed by the Topic Operator.
* Kafka users:: managed by the User Operator